### DISCUSSION TITLE
Open to more destination

### DISCUSSION THREAD

**Discussion by UnclePhil** [tiredofit/docker-db-backup#137] [Ideas]
Hello,  
I'm searching if someone have tried to inject rclone in the container to add more destinations for the backup. 
Because, not everyone have a S3 or a minio instance on his network.

I think rclone will be a perfect addendum.
and i think i'm going to fork and create it ... if it doesn't exist

Ph koenig

**Comment by tiredofit** [comment]
No experience on my end with Rclone, but happy to review a PR and merge and also assist with development / customization once its at a place that is usable..

**Reply by UnclePhil** [reply]
Hi Dave, 
have a look on this variant (https://github.com/UnclePhil/docker-db-backup_rclone) .
unclephil/docker-db-bbackup-rclone:0.0.3  on docker hub

It's based on your 3.6.1 image with some overwriting ( and some deletion)  

Working almost perfectly on my home infra. 
Just need some more tuning and your advise 
 
best regards, 
Ph koenig

**Reply by UnclePhil** [reply]
Suggestion. 
Can you split the 10_backup library in 2 parts.
- 10_dump with all db  export functions
- 10_upload with all upload functions.
In this  case anybody can overwrite the upload mechanism without loosing any db features. 

A 10_notify can be a must

Best regards

Ph koenig
